.github/workflows/ci.yml

```diff
@@ -0,0 +1,158 @@
+name: ci
+on:
+  push:
+  pull_request:
+    types: [opened, reopened]
+env:
+  PRODUCTION_BRANCH: refs/heads/release
+  STAGING_BRANCH: refs/heads/staging
+jobs:
+  ci:
+    name: Lint
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use Node.js
+        uses: actions/setup-node@v1
+        with:
+          node-version: '12.x'
+      - name: Cache Node.js modules
+        uses: actions/cache@v2
+        with:
+          # npm cache files are stored in `~/.npm` on Linux/macOS
+          path: ~/.npm
+          key: ${{ runner.OS }}-node-${{ hashFiles('**/package-lock.json') }}
+          restore-keys: |
+            ${{ runner.OS }}-node-
+            ${{ runner.OS }}-
+      - run: npm ci
+      - run: npx lockfile-lint --type npm --path client/package-lock.json --validate-https --allowed-hosts npm
+      - run: npx lockfile-lint --type npm --path server/package-lock.json --validate-https --allowed-hosts npm
```

Comment:
check twice means twice as secure issit

Reply:
Checking on `client` and `server`, since two package-locks

Reply:
thank god we have someone who actually knows how to review code

Reply:
OH WAIT DIFFERENT LOCKFILES I take back my snark

Reply:
wait it's cause one for `client` and one for `server`, that's why two lines